plugins/refetch-interval/src/index.ts

```diff
@@ -0,0 +1,141 @@
+import type { PiniaColadaPluginContext } from '@pinia/colada'
+import { toValue } from 'vue'
+/**
+ * @module @pinia/colada-plugin-refetch-interval
+ */
+
+/**
+ * Options for the Pinia Colada Refetch Interval plugin.
+ */
+export interface RefetchIntervalOptions {
+  /**
+   * If set to a number, queries will continuously refetch at this frequency in milliseconds.
+   * If set to false, no automatic refetching will occur.
```

Comment:
What about setting it to `true` or `'auto'` and have them auto refresh when stale? https://github.com/posva/pinia-colada/pull/97